pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -97,6 +97,10 @@
         %div
           %a{onclick: "return showVideo_mainvideo();", style: "cursor: pointer"}
             %img{src: "images/homepage/watch_video_drawn_button.png", style: "width: 250px; margin-top: 10px"}
+      - elsif entry[:type] == "celebrate_bhm"
+        %a.herohocbutton.bhmbutton{href: "/blackhistorymonth"}
```

Comment:
Visually, something looks a bit off?  The icon and text don't look quite aligned vertically and the icon might be too "bold"?  (Sorry for the nitpick, I used to be a newspaper layout editor.)  At first, I thought it was the font that was too small but I see that it's the same as the font that's currently being used for "Watch the video".  Maybe there's something we can adjust with the new icon?

Reply:
You are correct! I was missing a vertical-align: middle on that span of text, should be fixed now! Good eye